cli/wrappers/udf.ts

```diff
@@ -26,10 +26,6 @@ interface UdfRequestPayload {
   args: unknown
```

Comment:
Lines 3-4 can also be removed

Reply:
Line 25 should probably be modified since `kv` won't be passed anymore. We'd probably want it to be an empty object for now